workmanager/lib/src/workmanager.dart

```diff
@@ -179,31 +178,32 @@ class Workmanager {
   /// The [taskName] is the value that will be returned in the [BackgroundTaskHandler]
   /// The [inputData] is the input data for task. Valid value types are: int, bool, double, String and their list
   Future<void> registerOneOffTask(
-    /// Only supported on Android.
```

Comment:
Probably use default dart formatting `2-space indenting`, so that it does not change the code which is not modified